src/patternfly/components/Check/examples/Check.md

```diff
@@ -45,4 +71,5 @@ If you extend this component or modify the styles of this component, then make s
 | `.pf-c-check` | `<div>`, `<label>` |  Initiates the check component. **Required**  |
 | `.pf-c-check__input` | `<input type="checkbox">` |  Initiates a check input. **Required**  |
 | `.pf-c-check__label` | `<label>`, `<span>` |  Initiates a label. **Required**  |
-| `.pf-m-disabled` | `.pf-c-check__label` |  Initiates a disabled style for labels. **Required when input is disabled** |
\ No newline at end of file
+| `.pf-c-check__description` | `<p>` |  Initiates a check description. It is displayed vertically by default. |
```

Comment:
It's probably better to leave this as a `<div>` in case users want to use an inline list or something other than phrasing content.

Reply:
Also since we only have one layout for the description, I think we can remove the "It is displayed vertically by default" text.